app/javascript/components/server-components/TaxesPage.tsx

```
@@ -0,0 +1,667 @@
+              noCentsIfWhole: false,
+            })}
+          </td>
+          <td style={{ padding: "16px 12px", textAlign: "left", fontWeight: "500" }}>
+            {formatPriceCentsWithCurrencySymbol("usd", document.net_cents, {
+              symbolFormat: "short",
+              noCentsIfWhole: false,
+            })}
+          </td>
+          <td style={{ padding: "16px 12px" }}>
+            {(() => {
+              const { handleMouseEnter, handleMouseLeave } = useHoverEffect(2);
```

Biome (2.1.2)

[error] This hook is being called from a nested function, but all hooks must be called unconditionally from the top-level component.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)